pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -320,7 +320,7 @@ cdef class _Timestamp(datetime):
         cdef:
             int64_t val
             dict kwds
-            int8_t out[1]
+            ndarray[uint8_t, cast=True] out
```

Comment:
does this actually need the cast arg? we don't use this anywhere else IIRC

Reply:
please read the issue.

Reply:
we are not requiring cython >= 0.30 anytime soon, but I see for bools this is needed, ok

Reply:
thanks.